config/routes.rb

```diff
@@ -6,12 +6,12 @@
       resource :player, only: :show, controller: :podcast_player
       resources :imports
       resource :planner, only: [:show, :create], controller: :podcast_planner
-      resources :feeds, only: [:index]
+      resources :feeds, except: [:index, :edit]
       resources :episodes, only: [:index, :create, :new]
     end
 
     resources :episodes, except: [:create, :new]
-    resources :feeds
+    # resources :feeds
```

Comment:
do we want to keep the original `/feeds` routing? i'm not aware of the use case for it.

Reply:
Yeah, go ahead and remove.  I used to host an API at that endpoint, that Dovetail Router used.

But - it seems I already moved that to `/api/v1/feeds`.  And in staging right now, that `/feeds` path returns an html list.